modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/ObjectRelationshipInheritanceCheckbox.tsx

```diff
@@ -37,22 +45,29 @@ export function ObjectRelationshipInheritanceCheckbox({
 					alignPosition="top"
 					disableScroll
 					header={Liferay.Language.get('inheritance')}
+					onMouseLeave={() => setShowPopover(false)}
 					onShowChange={setShowPopover}
 					show={showPopover}
 					trigger={
 						<ClayIcon
 							className="field-base-tooltip-icon"
-							onBlur={() => setShowPopover(false)}
 							onFocus={() => setShowPopover(true)}
-							onMouseLeave={() => setShowPopover(false)}
 							onMouseOver={() => setShowPopover(true)}
 							symbol="question-circle-full"
 						/>
 					}
 				>
 					{Liferay.Language.get(
-						'enable-inheritance-to-share-settings-between-related-data-models'
+						'enable-inheritance-to-share-settings-across-related-data-models'
```

Comment:
hey @DavysonMelo, I believe you forgot to undo this change :sweat_smile:

Reply:
My bad 😅 I really forgot to change it

Reply:
Don't worry! Can you squash the last two fix commits into the commits that originated them? 😅